File: 19-02-2026/ski_mountaineering.py
def avalanche_risk(snow_depth, slope):
    
  lookupTable = {
    "Shallow": {
      "Steep": "Safe",
      "Very Steep": "Safe",
    },
    "Moderate": {
      "Steep": "Risky",
      "Very Steep": "Risky",
    },
    "Deep": {
      "Steep": "Risky",
      "Very Steep": "Risky",
    },
  };

  return "Safe" if slope == "Gentle" else lookupTable[snow_depth][slope];


print(avalanche_risk("Shallow", "Gentle")); # Safe
print(avalanche_risk("Moderate", "Steep")); # Risky
print(avalanche_risk("Deep", "Very Steep")); # Risky

File: 19-02-2026/ski_mountaineering.js
function avalancheRisk(snowDepth, slope) {
  const lookupTable = {
    Shallow: {
      Steep: "Safe",
      "Very Steep": "Safe",
    },
    Moderate: {
      Steep: "Risky",
      "Very Steep": "Risky",
    },
    Deep: {
      Steep: "Risky",
      "Very Steep": "Risky",
    },
  };

  return slope == "Gentle" ? "Safe" : lookupTable[snowDepth][slope];
}

console.log(avalancheRisk("Shallow", "Gentle")); // Safe
console.log(avalancheRisk("Moderate", "Steep")); // Risky
console.log(avalancheRisk("Deep", "Very Steep")); // Risky
